ksqldb-parser/src/main/java/io/confluent/ksql/parser/AstBuilder.java

```diff
@@ -725,6 +727,11 @@ public Node visitTerminateQuery(final SqlBaseParser.TerminateQueryContext contex
 
     @Override
     public Node visitShowColumns(final SqlBaseParser.ShowColumnsContext context) {
+      final String temp = context.sourceName().getText();
+      if (context.sourceName().identifier() instanceof SqlBaseParser.UnquotedIdentifierContext
```

Comment:
can you add a comment with some context about why we have a special case here because we don't want to break compatibility with previous statements that may have used TABLES to reference a column in an expression?